File: react-app/src/Componnents/features/lend/Lend.js


const Lend=({lend})=>{
    return(
        <>
        <h1>lend</h1>
        <h1>{lend.id}</h1>
        <h3>{lend.date}</h3>
        <h1>times</h1>
        <h3>{lend.takeHour}:00</h3>
        <h3>{lend.returnHour}:00</h3>
        </>
    )
}
export default Lend;

// public int ProductId { get; set; }
// public Product Product { get; set; }
// public int UserId { get; set; }
// public User User { get; set; }

File: c#/project/DAL/Lend.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace DAL
{
    public class Lend
    {
        public int ID { get; set; }
        public DateTime Date { get; set; }
        public int TakeHour { get; set; }   
        public int ReturnHour { get; set; }
        public int ProductId { get; set; }
        public virtual Product Product { get; set; }
        public int UserId { get; set; }
        public virtual User User { get; set; }

    }
}
